components/Carousel/Carousel.js

```diff
@@ -12,3 +6,24 @@ let carousel = document.querySelector();
     5. Think of how you would animate this component. Make the cards slide in and out, or fade. It's up to you!
     6. Have fun!
 */
+class Carousel {
+    constructor(carousel){
+        this.carousel = carousel;
+        this.leftButton = this.carousel.querySelector('.left-button');
+        this.rightButton = this.carousel.querySelector('.right-button');
+        this.imgs = this.carousel.querySelectorAll('[data-img]');
+        this.leftButton.addEventListener('click', this.leftButton());
```

Comment:
Your event listeners should look like:

this.leftButton.addEventListener('click', () => {
            this.rightButton();
            });
        this.rightButton.addEventListener('click', () => {
            this.leftButton();
        });